File: game-server/src/main/java/org/forfun/mmorpg/game/gm/message/ReqGmCommand.java
package org.forfun.mmorpg.game.gm.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import jforgame.socket.share.annotation.MessageMeta;
import jforgame.socket.share.message.Message;

@MessageMeta(cmd = 1)
public class ReqGmCommand implements Message {

	@Protobuf
	private String params;

	public String getParams() {
		return params;
	}

	public void setParams(String params) {
		this.params = params;
	}

}

File: game-server/src/main/java/org/forfun/mmorpg/game/gm/message/ReqGmCommand.kt
package org.forfun.mmorpg.game.gm.message

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf
import jforgame.socket.share.annotation.MessageMeta
import jforgame.socket.share.message.Message

@MessageMeta(cmd = 1)
class ReqGmCommand : Message {

    @Protobuf
    var params: String? = null
}
